### PR TITLE
add tag build in ci

### PR DESCRIPTION
**Pull request by liam-lai**

### CODE REVIEW

File: .github/workflows/ci.yml
Comment (wanwiset25) [inline .github/workflows/ci.yml:1]: I think this is missing
`    tags:
      - '*'`

Comment (liam-lai) [reply .github/workflows/ci.yml:1]: done, thanks for review

Comment (wjrjerome) [reply .github/workflows/ci.yml:1]: hmm... this would trigger build of images for all tags no? or does this condition is on `branch=master AND tag=*`?

Comment (liam-lai) [reply .github/workflows/ci.yml:1]: @wjrjerome  no, so it's `or`

**Review by wanwiset25 — APPROVED** [review]

File: .github/workflows/ci.yml
Comment (wjrjerome) [reply .github/workflows/ci.yml:1]: ok. have u tried create a tag from any of the branch? does it trigger the build somehow? @liam-lai

Comment (liam-lai) [reply .github/workflows/ci.yml:1]: yes I have tried. If you tag, then build will run and tag the docker image.

**Review by wjrjerome — APPROVED** [review]

**Pull request merged**